src/qt/sendcoinsdialog.cpp

```diff
@@ -186,6 +187,13 @@ void SendCoinsDialog::setModel(WalletModel *_model)
         // set default rbf checkbox state
         ui->optInRBF->setCheckState(Qt::Checked);
 
+        if (model->privateKeysDisabled()) {
+            ui->sendButton->setText(tr("Create Unsigned"));
+            ui->sendButton->setToolTip(tr("Creates a Partially Signed Bitcoin Transaction (PSBT) for use with e.g. an offline Bitcoin Core wallet, or a PSBT-compatible hardware wallet."));
+        } else {
+            ui->sendButton->setText(tr("Send"));
```

Comment:
Do we need this? Each wallet gets its own `SendCoinsDialog`, and can't change `privateKeysDisabled` AFAIK...

Reply:
Not sure what you mean here. We set these strings in `setModel(WalletModel *_model)`. That's the first time we know `privateKeysDisabled`. The constructor doesn't know.

Reply:
I mean there is never a case where we need to change the text *back* to "Send" (which is set in the .ui)

Reply:
Ah I see. Ok, I dropped the else branch (keeping the original in the `.ui` file).